pkg/arvo/app/publish.hoon

```diff
@@ -962,41 +976,98 @@
     =/  pax=path  /app/publish/notebooks/[book.act]/publish-info
     :_  state
     [(write-file pax %publish-info !>(new-book)) cards]
+  ::  %new-note
   ::
       %new-note
```

Comment:
Having a bit of trouble parsing b990140. `%new-note` and `%new-comment` flows don't seem to be touching `limbo` at all. Is this correct? If yes, then why were these sections changed? It's not immediately apparent to me.

(It's a bit late here, forgive me if this should be obvious. But also a couple more comments here wouldn't be amiss. These sections feel pretty long.)

Reply:
Does my answer to your question above explain this sufficiently? I can go into more detail. I will add some comments too.

Reply:
Yeah I think that explains it. Comments would still be good. Thanks! (: